<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceWidget</class>
 <widget class="QWidget" name="FindReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>76</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>192</red>
        <green>191</green>
        <blue>188</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>192</red>
        <green>191</green>
        <blue>188</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>192</red>
        <green>191</green>
        <blue>188</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>192</red>
        <green>191</green>
        <blue>188</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="5">
    <widget class="QCheckBox" name="chkWhole">
     <property name="text">
      <string> Whole Words</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Replace with:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="ClickableLabel" name="btnFindNext">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="ClickableLabel" name="btnFindPrev">
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="ClickableLabel" name="btnReplaceAll">
     <property name="text">
      <string>Replace all </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="chkCaseSensitive">
     <property name="text">
      <string>Case Sensitive</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ClickableLabel" name="btnReplace">
     <property name="text">
      <string>Replace </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtReplace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="ClickableLabel" name="btnReplaceAndFind">
     <property name="text">
      <string>Replace &amp; Find </string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="ClickableLabel" name="btnSelectAll">
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtFind">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="ClickableLabel" name="btnHide">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
